### DISCUSSION TITLE
My Profile and Comments not visible to others

### DISCUSSION THREAD

**Discussion by rohailtaha** [community/community#58004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

All of a sudden, my pull requests and comments are not visible anymore to other members/collaborators on a private repository. I myself can see my comments and PRs. I am also not visible in the collaborators list although I was visible before and I have made significant contributions to the repo. I am still able to perform code actions like merge PR, create PR, add comments etc. 
Also my profile page https://github.com/rohailtaha appears as 404 to others but not myself. But When I opened my profile page link on incognito mode, I can see the 404 page.

![image](https://github.com/community/community/assets/55291205/0379085c-8518-4655-8cff-196c4061d84d)

Another thing is that when I **_react_** to a comment using a thumbs up or other emoji, this reaction is visible to others.

![image](https://github.com/community/community/assets/55291205/bbdb697e-cd84-4635-a432-278b0c871d2c)

I don't know the solutions but I have tried:
- Enabling two factor auth
- Changing my password

This issue suddenly happened yesterday and I haven't done anything with my account recently. Everything was working fine before. Is my account blocked to others? Why my profile is 404? I would really appreciate the help and as soon as possbile as I really need to work on that repo.

**Comment by SpaxscE** [comment]
how did you fix it?

**Comment by Ashokininternet** [comment]
Your profile : https://github.com/rohailtaha is visible now.
Ig i was a temporary issue or a bug that you profile was no visible so now its fixed ⚙.
Let me know things on your side😁.

Edit:
Btw if this was helpful can you mark it as answer 😋.

**Reply by monsieurtarlan** [reply]
> Your profile : https://github.com/rohailtaha is visible now. Ig i was a temporary issue or a bug that you profile was no visible so now its fixed ⚙. Let me know things on your side😁.
> 
> Edit: Btw if this was helpful can you mark it as answer 😋.

my profile is also not visible. can you help me?
